axelor-account/src/main/java/com/axelor/apps/account/service/payment/PaymentModeServiceImpl.java

```diff
@@ -78,6 +78,17 @@ public Account getPaymentModeAccount(
         paymentMode.getName());
   }
 
+  @Override
+  public Account getPaymentModeAccount(PaymentMode paymentMode, Company company)
+      throws AxelorException {
```

Comment:
I'm not sure about this way. I think we have to get the account for the correct paymentMode, company AND the bankDetails because we can have two accountManagement lines with same company but different bankDetails and different account.